### ISSUE TITLE
Need Support: Local Model Parameters Override Like Llama.cpp

### ISSUE THREAD

**Issue by DirtyKnightForVi** [ollama/ollama#4904]
In llama.cpp, when running a model, you can update the model parameters using `--override-kv`. How can this be achieved in ollama? Should you modify a certain file? Add `PARAMETER` to the Modlefile? Or is there some other similar command?

```
Metadata KV overrides (pass them using --override-kv, can be specified multiple times):

deepseek2.attention.q_lora_rank=int:1536
deepseek2.attention.kv_lora_rank=int:512
deepseek2.expert_shared_count=int:2
deepseek2.expert_feed_forward_length=int:1536
deepseek2.expert_weights_scale=float:16
deepseek2.leading_dense_block_count=int:1
deepseek2.rope.scaling.yarn_log_multiplier=float:0.0707
```

**Comment by kanishkaditya** [comment]
hey any way to pass kv overrides to ollama yet?
i have been puzzeled whole day due to this, specifically i want to increase the number of expert but its a kv overide. Not sure how to do it yet.

**Comment by rick-github** [comment]
> hey any way to pass kv overrides to ollama yet?

No.  My change languished without being merged so I closed it.  The runners that are now built for ollama no longer support the `--override-kv` flag.

**Comment by ProjectMoon** [comment]
What is the best way to achieve the overrides then? Is it possible to set values in the config json when importing from safe tensors?

**Comment by rick-github** [comment]
I expect that modifying the config.json and building a new model would work.  The runner architecture that ollama uses is undergoing a lot of changes at the moment, when the transition to the new system is complete there may be an opportunity to add a KV override mechanism.  Until then, modifying the model is the only way to achieve this.
